install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -676,3 +676,8 @@ DELETE FROM `PREFIX_hook` WHERE name = 'actionFrontControllerAfterInit';
 
 /* Update wrong hook alias */
 UPDATE `PREFIX_hook_alias` SET name = 'displayHeader', alias = 'Header' WHERE name = 'Header' AND alias = 'displayHeader';
+
+/* add id_order in cart */
+ALTER TABLE `PREFIX_cart`
```

Comment:
Must be move in `1.7.7.1.sql`.

Reply:
nice catch ! 

it was not supposed to be 1.7.7.1 when I started this issue...